polkadot/xcm/pallet-xcm/src/benchmarking.rs

```diff
@@ -74,6 +74,13 @@ pub trait Config: crate::Config {
 	fn set_up_complex_asset_transfer() -> Option<(Assets, u32, Location, Box<dyn FnOnce()>)> {
 		None
 	}
+
+	/// Gets an asset that can be handled by the AssetTransactor.
+	///
+	/// Used only in benchmarks.
+	///
+	/// Used, for example, in the benchmark for `claim_assets`.
+	fn get_valid_asset() -> Asset;
```

Comment:
@franciscoaguirre 
We have already something similar `fn get_asset() -> xcm::latest::Asset;` for fungible benchmarks: https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/pallet-xcm-benchmarks/src/fungible/mod.rs#L47

Both have very similar docs:
```
/// Gets an asset that can be handled by the AssetTransactor.
fn get_valid_asset() -> Asset;

...
/// Give me a fungible asset that your asset transactor is going to accept.
fn get_asset() -> xcm::latest::Asset;
```

So the target runtime needs to provide "very similar/exact" function overrides, e.g.:

this new:
```
				fn get_valid_asset() -> Asset {
					Asset {
						id: AssetId(Location::parent()),
						fun: Fungible(ExistentialDeposit::get()),
					}
				}
```

and older one: https://github.com/paritytech/polkadot-sdk/pull/3403/files#diff-a3b656e8095fe815cf8611c8319aff7b2f32482cdbe7c24ac37a2ce45235f32dR1544-R1549
```
				fn get_asset() -> Asset {
					Asset {
						id: AssetId(TokenLocation::get()),
						fun: Fungible(UNITS),
					}
				}
```

I would say we should simplify that.

Other question, shouldn't this new benchmark `claim_assets` better use `claimable_asset` ?

Reply:
`claimable_asset` gives me the location of the user but not its FRAME origin. I don't have access to the converter from Location to Origin in the benchmark I think.